addon/pods/components/frost-tab/template.hbs

```diff
@@ -1 +1,15 @@
-{{yield}}
+<div class='tab' data-test={{hook (concat hook '-tab' (if isSelected '-selected' '')) }}>
+  {{frost-button
+    class=(concat 'tab' (if isSelected ' active'))
```

Comment:
Hmm I know it works in html tags directly. Maybe its not supported in a component in that way.